articles/cognitive-services/Speech-Service/includes/how-to/conversation-transcription/real-time-csharp.md

```diff
@@ -107,6 +107,7 @@ This sample code does the following:
 * Creates a `ConversationTranscriber` using the constructor, and subscribes to the necessary events.
 * Adds participants to the conversation. The strings `voiceSignatureStringUser1` and `voiceSignatureStringUser2` should come as output from the steps above from the function `GetVoiceSignatureString()`.
 * Joins the conversation and begins transcription.
+* If you want to differentiate speakers but no voice samples input, please enables `DifferentiateGuestSpeakers` feature as in [Conversation Transcription Overview](../../../conversation-transcription.md). 
```

Comment:
>If you want to differentiate speakers but no voice samples input, [](http://example.com/codeflow?start=2&length=65)

If you want to differentiate speakers without providing voice samples,

Reply:
Updated~